### ISSUE TITLE
transfer failed: commission write-off but coins not sent

### ISSUE THREAD

**Issue by tina80lvl** [toncenter/ton-wallet#72]
Can't transfer toncoin to another wallet. Tried a few times, but you **wallet takes only commission** from me, not transferring coins. COINS ARE GONE. Process just freezes, attaching video. After that everything freezing

https://user-images.githubusercontent.com/27859234/149594398-1d6c51f5-294e-4b66-92a7-771c0ee3ee01.mp4

**Comment by ubaystian** [comment]
Hi, same on me.. have you get this issue solved?

**Comment by tina80lvl** [comment]
No... Tried once again today, lost fee AGAIN and didn't get coins, AGAIN. Look at the time and amount @tolya-yanot @neverthirty @EmelyanenkoK 
![IMG_0914](https://user-images.githubusercontent.com/27859234/151702763-a6ee5b4f-6374-4a2f-8501-1d8eb5a4df56.jpeg)
![IMG_0915](https://user-images.githubusercontent.com/27859234/151702766-696bec35-6a96-4a6b-8695-e59ad69da9df.jpeg)
![IMG_0916](https://user-images.githubusercontent.com/27859234/151702770-14cfadd4-a700-48fd-802c-c65c27921e45.jpeg)

**Comment by appFuckthebashed** [comment]
наебалово!

сб, 15 янв. 2022 г., 3:39 Tina ***@***.***>:
